app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/SearchResultsAdapter.java

```diff
@@ -53,7 +53,7 @@ public RssViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
     @Override
     public void onBindViewHolder(final RssViewHolder holder, int position) {
         Datum datum = datumList.get(position);
-        if (datum != null) {
```

Comment:
Remove the second line `holder.titleTextView.setText(Html.fromHtml(datum.getTitle()));` .Why have you written this? It is already written in else.